db/migrate/20160110104053_add_claim_ref_to_container_volumes.rb

```diff
@@ -0,0 +1,13 @@
+class AddClaimRefToContainerVolumes < ActiveRecord::Migration
+  def up
+    change_table :container_volumes do |t|
+      t.belongs_to :persistent_volume_claim, :type => :bigint
+    end
+  end
+
+  def down
+    change_table :container_volumes do |t|
+      t.remove_belongs_to :persistent_volume_claim
+    end
+  end
```

Comment:
Minor, but if you are using change_table, you don't need to define both the up and down...


```ruby
def change
  change_table :container_volumes do |t|
    t.belongs_to :persistent_volume_claim, :type => :bigint
  end
end
```